src/Type/BaseDoctrineORMSerializationType.php

```diff
@@ -63,7 +65,7 @@ class BaseDoctrineORMSerializationType extends AbstractType
      * @param MetadataFactoryInterface $metadataFactory     Serializer metadata factory
      * @param ManagerRegistry          $registry            Doctrine registry
      * @param string                   $name                Form type name
-     * @param string                   $class               Data class name
+     * @param class-string             $class               Data class name
```

Comment:
I think this one should be added as phpstan-param

Reply:
Done